api/document/json_importer/annotations.py

```diff
@@ -76,6 +74,16 @@ def derive_annotations(cursor: JSONAwareCursor) -> AnnotationDict:
     return annotations
 
 
+def get_content_length(content: List[PrimitiveDict]) -> int:
+    length = 0
+    for c in content:
+        if c['content_type'] == '__text__':
+            length += len(c['text'])
+        else:
+            length += get_content_length(c['inlines'])
+    return length
```

Comment:
This is _almost_ identical to `get_content_text`. Should we just run `len(get_content_text(content))`?

Reply:
Yeah, I was probably over-optimizing--that was originally what I was going to do, but I didn't want to require python to allocate a bunch of strings only to not use them. But it's also a bunch of added code that needs to be separately tested and whatnot. Blech.